components/ui/pathogen-map/use-pathogen-map-mouse.ts

```diff
@@ -0,0 +1,68 @@
+import { useState } from "react";
```

Comment:
All this `cursor`, `setCursor`, `onMouseEnter`, `onMouseLeave` stuff is for is so that the cursor has the right icon when you hover over a point on the map.

Docs: https://visgl.github.io/react-map-gl/docs/api-reference/map#onmouseenter